tests/functional/monitoring/prometheus/metrics/test_ocs_exporter_deployment.py

```
@@ -0,0 +1,340 @@
+    logger.info("✓ Deployment spec has no kube-rbac-proxy image references")
+
+    # Verify service configuration
+    from ocs_ci.ocs.ocp import OCP
+
+    ocp_service = OCP(kind=constants.SERVICE, namespace=namespace)
+    services = ocp_service.get(selector=constants.OCS_METRICS_EXPORTER).get("items", [])
+
+    if services:
+        service = services[0]
+        ports = service.get("spec", {}).get("ports", [])
+        logger.info(f"Service ports: {ports}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)